spec/integration/manifest.spec.yml

```diff
@@ -487,6 +500,25 @@ spec:
     - version
     - type
     - owner
+  allOf:
+    - if:
+        properties:
+          policy_templates:
+            maxItems: 1
+      then:
+        properties:
+          policy_templates_behavior:
+            enum:
+              - default
+            default: default
```

Comment:
Maybe we could even disable the option if there is a single policy template?

Reply:
I tried to forbid/disable that option in case of a single policy template, with this conditional schema:
```yaml

  allOf:
    - if:
        properties:
          policy_templates:
            maxItems: 1
      then:
        # not need to define policy_templates_behaviour if there is just one policy template
        not:
          required:
            - policy_templates_behavior
```

But the error was not descriptive enough I think to help to debug the error:
> field (root): Must not be present